docs/api/native-image.md

```diff
@@ -137,6 +137,18 @@ let image = nativeImage.createFromPath('/Users/somebody/images/icon.png')
 console.log(image)
 ```
 
+### `nativeImage.createFromBitmap(buffer, options)`
+
+* `buffer` [Buffer][buffer]
+* `options` Object
+  * `width` Integer
+  * `height` Integer
+  * `scaleFactor` Double (optional) - Defaults to 1.0.
+
+Returns `NativeImage`
+
+Creates a new `NativeImage` instance from `buffer`.
```

Comment:
This should describe the expected pixel format. Is it RGBA8888? Or some other form?

Reply:
seems to be platform specific (BGRA8888 or RGBA8888). IMO saying it's the raw bitmap pixel data as returned by `toBitmap()` should be enough. Otherwise we need to document what `toBitmap()` returns as well.